tests/rust/re_integration_test/tests/origin_heuristics_test.rs

```diff
@@ -0,0 +1,246 @@
+use egui_kittest::kittest::Queryable as _;
```

Comment:
Should have a module-level comment explaining roughly what this test is all about